python/python/glide/config.py

```diff
@@ -284,3 +289,13 @@ def __init__(
             client_name=client_name,
             protocol=protocol,
         )
+        self.periodic_checks_interval = periodic_checks_interval
+
+    def _create_a_protobuf_conn_request(
+        self, cluster_mode: bool = False
```

Comment:
```suggestion
        self, cluster_mode: bool = True
```
Why `False`? It is `ClusterClientConfiguration` as I understand

Reply:
It's the base function signature